tests/pytests/test_viking_drivers.py

```diff
@@ -0,0 +1,302 @@
+                                                -0.011566569536285, -0.9999824886402,
+                                                0.0058586590004418, -8.35779681749578e-04,
+                                                -7.67944795396292e-04, 0.011571271291668,
+                                                0.99993275570985],
+                           'CkTableStartTime': -676709864.3045,
+                           'CkTableEndTime': -676709864.3045,
+                           'CkTableOriginalSize': 1,
+                           'EphemerisTimes': [-676709864.3045],
+                           'Quaternions': [[0.14671376032888,-0.20207775752139,0.96766874650245,0.03545207310812]],
+                           'AngularVelocity': [[0.0,0.0,0.0]]},
+    'BodyRotation': {'TimeDependentFrames': [10014, 1],
+                      'CkTableStartTime': -676709864.3045,
+                      'CkTableEndTime': -676709864.3045,
+                      'CkTableOriginalSize': 1,
+                      'EphemerisTimes': [-676709864.3045],
+                      'Quaternions': [[-0.028648237381698,0.24168074000221,-0.20688348409661,0.94761222154157]],
+                      'AngularVelocity': [[3.16266051008033e-05,-2.87737768090931e-05,5.6534158586979e-05]]},
+    'InstrumentPosition': {'SpkTableStartTime': -676709864.3045,
+                           'SpkTableEndTime': -676709864.3045,
+                           'SpkTableOriginalSize': 1.0,
+                           'EphemerisTimes': [-676709864.3045],
+                           'Positions': [[3297.8743337482,966.46050377782,10725.740343691]],
+                           'Velocities': [[-0.44211958524766,1.2883896612093,1.9012944589374]]},
+    'SunPosition': {'SpkTableStartTime': -676709864.3045,
+                    'SpkTableEndTime': -676709864.3045,
+                    'SpkTableOriginalSize': 1.0,
+                    'EphemerisTimes': [-676709864.3045],
+                    'Positions': [[222364052.49946,87428117.498071,34077452.005079]],
+                    'Velocities': [[-10.327617595005,18.315940117221,8.6804664968805]]}}}
+    }
+
+@pytest.fixture(scope="module")
+def test_kernels():
+    updated_kernels = {}
+    binary_kernels = {}
+    for image in image_dict.keys():
+        kernels = get_image_kernels(image)
+        updated_kernels[image], binary_kernels[image] = convert_kernels(kernels)
+    yield updated_kernels
+    for kern_list in binary_kernels.values():
+        for kern in kern_list:
+            os.remove(kern)
+
+@pytest.mark.parametrize("label_type", ['isis3'])
+@pytest.mark.parametrize("formatter", ['isis'])
+@pytest.mark.parametrize("image", image_dict.keys())
+def test_viking1_load(test_kernels, label_type, formatter, image):
+    label_file = get_image_label(image, label_type)
+    isis_isd = ale.load(label_file, props={'kernels': test_kernels[image]}, formatter=formatter)
+    print(isis_isd)
```

Comment:
Already merged, but maybe get rid of print statement?

Reply:
We've been leaving in the isd print statements because if the test fails it can be helpful to have the output for ALE. If the test passes, output is not printed to the terminal.